Update usage-life-cycle.md (#375)
BitcoinDesign/Guide: guide/designing-products/usage-life-cycle.md

# Usage life cycle
{:.no_toc}

As we develop new skills, things that are difficult at first eventually become intuitive to us. So when we develop products, it’s a good idea to regularly step back and look at the products we create through users' eyes on that same journey.

> The following six sections outline a typical usage life cycle, from never having heard of your product to becoming one of its power users.

Reviewer: ```suggestion
The following six phases outline a typical usage life cycle, from never having heard of your product to becoming one of its power users.
```
We use sections to refer to parts of the guide, so changing this to phases, which is also used in the following paragraph.

   layout = "float-left"
%}

> The user decides to move forward with your product. They are still not familiar with exactly how it works, and their expectations may not match what your product offers. Therefore, it is crucial to tightly manage the [onboarding experience research]({{ '/guide/onboarding/introduction/' | relative_url }}). Your product should make it easy for users to quickly and successfully solve problems while being introduced to your product's basics and best practices. This sets expectations for future interactions. If the first use is frustrating, users are less likely to use your product again—or to recommend it.

Reviewer: ```suggestion
The user decides to move forward with your product. They are still not familiar with exactly how it works, and their expectations may not match what your product offers. Therefore, it is crucial to pay close attention to the design of the [onboarding experience]({{ '/guide/onboarding/introduction/' | relative_url }}). Your product should make it easy for users to quickly and successfully solve problems while being introduced to your product's basics and best practices. This sets expectations for future interactions. If the first use is frustrating, users are less likely to use your product again—or to recommend it.
```
Not sure what is meant by `tightly managing the onboarding experience research`. It's the design of the experience that needs to be well-crafted.

   layout = "float-right-desktop"
%}

Expects to lose users at every step in the life cycle. This is unavoidable. If you understand where and why they drop off, you can at least reduce the likelihood of it happening by solving for the issues that cause dropoff.

> Barious [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify and solve these types of issues

Reviewer 2: ```suggestion
Various [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify and solve these types of issues
```

Author: rofl

> Various [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify and solve these types of issues

Reviewer: ```suggestion
There are various [user research]({{ '/guide/designing-products/user-research/' | relative_url }}) techniques that can help identify and solve these types of issues.
```
Sentence had no verb or period.
